spaceship/lib/spaceship/du/upload_file.rb

```diff
@@ -37,6 +37,25 @@ def remove_alpha_channel(original)
         `sips -s format png '#{path}'`
         return path
       end
+
+      # Construct proper filename for upload file, containing md5 hash of the file's content and it's original filename
+      # @param md5 (String) md5 of files content
+      # @param path (String) path to a file
+      # @return (String)
+      def construct_upload_filename(md5, path)
+        'ftl_' + md5 + '_' + File.basename(path)
```

Comment:
Maybe we want some very basic input check here, since this line would crash if either md5 or path is nil